### PR TITLE
Jhonson Algorithm in C#

### LINKED ISSUE #2206: Johnson Algorithm
- [x] C 
- [ ] C++ - @Aakansha99
- [x] C# - @ritish099
- [ ] CoffeeScript
- [x] Dart - @Swarnimashukla
- [ ] Go - @BhardwajShweta25
- [ ] Java - @Neelaksh-Singh
- [ ] JS - @thakran14
- [x] Kotlin - @ritish099
- [x] Php - @ritish099
- [ ] Python - @BishalKrChanda
- [ ] Readme - @AdityaSingh17
- [ ] Ruby
- [ ] Rust
- [ ] TypeScript

### PR DESCRIPTION
**Pull request by ritish099**
Fixes #2206

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Added file for Jhonson Algorithm
- 

#### Languages Used:
- C#

#### Files Added:
- Jhonson.cs

Thanks!

### CODE REVIEW

**Comment by ritish099** [comment]
@MastersAbh @asha15 @SakshiShreya review it

File: Jhonson_Algoritm/Jhonson.cs [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+namespace ConsoleApp1
+{
+    class Jhonson_Algorithm
+    {
+        public static void Main(String[] args)
+        {
+            int vert, edge, i, j, k, c;
+            int INF = 999999;
+            int[,] cost = new int[10, 10];
+            int[,] adj = new int[10, 10];
```
Comment (asha15) [inline Jhonson_Algoritm/Jhonson.cs]: keep a line space after this

File: Jhonson_Algoritm/Jhonson.cs [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+            Console.WriteLine("Enter no of vertices: ");
+            vert = Convert.ToInt32(Console.ReadLine());
+            Console.WriteLine("Enter no of Edges: ");
+            edge = Convert.ToInt32(Console.ReadLine());
+            Console.WriteLine("Enter the EDGE cost: ");
```
Comment (asha15) [inline Jhonson_Algoritm/Jhonson.cs]: keep a line space after this

File: Jhonson_Algoritm/Jhonson.cs [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+            for (k = 1; k <= edge; k++)
+            {
+                //take the input and store it into adj and cost matrix
+                i = Convert.ToInt32(Console.ReadLine());
+                j = Convert.ToInt32(Console.ReadLine());
+                c = Convert.ToInt32(Console.ReadLine());
+                adj[i, j] = cost[i, j] = c;
+            }
```
Comment (asha15) [inline Jhonson_Algoritm/Jhonson.cs]: keep a line space after this

File: Jhonson_Algoritm/Jhonson.cs [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+                for (j = 1; j <= vert; j++)
+                {
+                    if (adj[i, j] == 0 && i != j)
+                        adj[i, j] = INF; //If its not a edge put infinity
+                }
+            for (k = 1; k <= vert; k++)
+                for (i = 1; i <= vert; i++)
+                    for (j = 1; j <= vert; j++)
+                        //Finding the minimum
+                        adj[i, j] = (adj[i, k] + adj[k, j]) > adj[i, j] ? adj[i, j] : (adj[i, k] + adj[k, j]) ; //find minimum path from i to j through k
```
Comment (asha15) [inline Jhonson_Algoritm/Jhonson.cs]: can you bring the commend to the previous line

**Pull request merged**